http-clients-instrumentation/src/jvmMain/kotlin/com/epam/drill/agent/instrument/TransformStrategy.kt

```diff
@@ -16,14 +16,11 @@
 package com.epam.drill.agent.instrument
 
 import com.epam.drill.agent.instrument.util.*
-import com.epam.drill.logger.*
 import javassist.*
 import java.security.*
 
 abstract class TransformStrategy : IStrategy {
 
-    protected val logger = Logging.logger { this::class.java.name }
```

Comment:
There is no new logger for abstract class (I guess we rely on the actual implementation now, right?)